mapie/regression/time_series_regression.py

```diff
@@ -454,16 +461,20 @@ def predict(
               - [:, 0, :]: Lower bound of the prediction interval.
               - [:, 1, :]: Upper bound of the prediction interval.
         """
-        if alpha is None:
```

Comment:
Unsure about this refactoring of the .predict method, starting with the added `return`.